docs/vhost-user-net-testing.md

```diff
@@ -0,0 +1,126 @@
+# Vhost-user-net
+
+The purpose of this document is to illustrate how to test vhost-user-net in cloud-hypervisor with OVS/DPDK as backend.
+
+## Framework
+
+It's a simple test to validate the communication between two virtual machine, connecting them to vhost-user ports respectively provided by `OVS/DPDK`.
```

Comment:
is "vhost-user ports" the right terminology?

Reply:
Yes, I think so.